File: Daily Coding/c/sum_of_prime.c
/*
Given a number, the problem has to print two prime numbers whose sum will be equal to given number.
*/
#include <stdio.h>
int is_prime(int num)
{
    int i;
    for(i=3; i<=(num/2); i++)
    {
        if(num%i == 0)
            return 0;
    }
    return 1;
}
int main()
{
    int num, i;
    printf("Enter the number: ");
    scanf("%d",&num);
    for(i=3; i<=num; i++)
    {
        if(is_prime(i) && is_prime(num-i))
        {
            printf("%d %d",i,num-i);
            break;
        }
    }
    return 0;
}

File: Daily Coding/python/sum_of_prime.py
"""
Given a number, the problem has to print two prime numbers whose sum will be equal to given number.
"""
def is_prime(num):
    for i in range(3,(num//2)+1):
        if num%i == 0:
            return False
    return True

num = int(input("Enter the number: "))
for i in range(3,num+1):
    if is_prime(i) and is_prime(num-i):
        print(i,num-i)
        break
